style.css

```diff
@@ -762,7 +767,177 @@ input[type=submit]:disabled {
   height: 100vh;
 }
 
+.modal {
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0);
+  backdrop-filter: blur(6px);
+  position: absolute;
+  display: none;
+  justify-content: center;
+  align-items: center;
+  top: 0%;
+  left: 0%;
+  z-index: 9999;
+}
+
+.modal-content {
+  top: 0%;
+  left: 0%;
+  width: 450px;
+  height: 92%;
+  overflow-y: scroll;
+  overflow-x: hidden;
```

Comment:
- [ ] Nice job on your mobile version 👏 . However, the changes you implemented as suggested by the last reviewer have created an added margin space on the right side of your mobile design.
- This does not give a better visual view and I advise you to fix this.

- Possible solution is to check for Overflow. Verify if any elements within the design are causing overflow or exceeding the available space e.g elements with fixed widths. Make necessary adjustments to prevent overflow and ensure everything is within the visible area. You can also check Media Queries. Utilize media queries in your CSS to specifically target and modify the layout for mobile devices. By adjusting the styles for smaller screens, you can eliminate the excess margin and optimize the visual view.

## Your view 👍 
![image](https://github.com/YoutLone/Portfolio/assets/50990925/3ebcc818-a374-41c8-9a97-f3bbc80d6e66)